strFunctions.js

```diff
@@ -1,7 +1,17 @@
+const ExtensionUtils = imports.misc.extensionUtils;
+const extension = ExtensionUtils.getCurrentExtension();
+const convenience = extension.imports.convenience;
+
+const Schema = convenience.getSettings(extension, 'hijri-calendar');
+
 function format(str) {
     let enums = ['0', '1', '2', '3', '4', '5', '6', '7', '8', '9'];
     let hnums = ['٠', '١', '٢', '٣', '٤', '٥', '٦', '٧', '٨', '٩'];
 
+    if (Schema.get_boolean('arab-numerals')) {
```

Comment:
And I think this should be reversed, no?
Currently, when the checkbox is checked, to “Display Arab numerals” it shows Latin numerals!
Or you are referring to Eastern/Western Arabic numbers?

Reply:
Yeah I see the confusion here!
Actually the most used numeric system in the world (0,1,2,3...) are called Arabic numerals while the symbols used in the eastern Arab region (٠،١،٢،٣...) are called Hindu-Arabic numerals.
Here are their respective Wikipedia pages:
- [Arabic numerals](https://en.wikipedia.org/wiki/Arabic_numerals)
- [Eastern Arabic numerals / Arabic-Hindu numerals](https://en.wikipedia.org/wiki/Eastern_Arabic_numerals)

Reply:
Then I would suggest to use "eastern/western" Arabic numbers and a maybe select box instead of a checkbox.
Or we can even avoid naming and put some examples only, like `0,1,2,3...` or `٠،١،٢،٣...`
Or both